README.md

```diff
@@ -1,30 +1,32 @@
 # mapsforgesrv
+when an old java 8 is installed, this branch is to be used for the development
```

Comment:
@nono303: the wording "this branch" is a bit confusing.

shouldn't we write: 

`When an old java 8 is installed, the branch "Java8" is to be used for the development`

Reply:
i think this is just a copy past error...

i found the same text later, where it make more sense ;-)

Reply:
Yes... must be a copy paste mistake as I didn't update `readme` , just back-port changes of https://github.com/telemaxx/mapsforgesrv/pull/15